src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportOrderCheckTest.java

```diff
@@ -782,4 +786,23 @@ public void testTrimOption() throws Exception {
                 getPath("InputImportOrderTestTrimInOption.java"),
                 expected);
     }
+
+    @Test
+    public void testClearState() throws Exception {
```

Comment:
Please do summary of my comment from PR to explain reason why we allow pure unit test.
We have a rule that all pure unit test must have explanation comment.

Reply:
Please add this case to our issue that collect bad sude effects of pitest - extra code that does help to make code more robust to future changes that might miss extended testing.

Reply:
Done